src/angularComponent/ngcOmniboxController.js

```diff
@@ -200,6 +200,19 @@ export default class NgcOmniboxController {
     }
   }
 
+  /**
+   * Removes a particular suggestion item frmo the list of selected items.
```

Comment:
Clears it out https://github.com/UrbanCompass/ngc-omnibox/pull/17/files/62ddcc5d36d05d2ed77982e56277c50ad228ca02#diff-d8a178e25f29a787558002bb505a778dR212